packages/myst-cli/src/build/typst.ts

```diff
@@ -261,12 +262,13 @@ export async function localArticleToTypstTemplated(
     const includeFileBases = results.map((result, ind) => {
       const base = `${name}-${content[ind]?.slug ?? ind}${ext}`;
       const includeFile = path.format({ dir, ext, base });
-      writeFileToFolder(includeFile, result.value);
+      const exports = renderTypstImports(false, collected);
```

Comment:
Ugh, a bit tedious we have to add these here, but they get added to the main page during render... but makes sense.

Reply:
Yeah, I was a bit suprised.